drivers/gles3/storage/render_scene_buffers_gles3.cpp

```diff
@@ -50,10 +51,14 @@ void RenderSceneBuffersGLES3::configure(RID p_render_target, const Size2i p_inte
 	//msaa = p_msaa;
 	//screen_space_aa = p_screen_space_aa;
 	//use_debanding = p_use_debanding;
-	//view_count = p_view_count;
+	view_count = p_view_count;
 
 	free_render_buffer_data();
 
+#if !defined(GLES_OVER_GL) && !defined(WEB_ENABLED) && !defined(IOS_ENABLED)
+	PFNGLFRAMEBUFFERTEXTUREMULTIVIEWOVRPROC glFramebufferTextureMultiviewOVR = config->eglFramebufferTextureMultiviewOVR;
+#endif
```

Comment:
I think this would look better as a macro at the top of the file like how we already handle ``glClearDepth`` -> ``glClearDepth`` on GLES3 devices. We could do:
```
#define glFramebufferTextureMultiviewOVR GLES3::Config::get_singleton()->eglFramebufferTextureMultiviewOVR
```

Reply:
Sure, can do!